PowerPointLabs/PowerPointLabs/Ribbon1.xml

```diff
@@ -188,11 +188,11 @@
                getLabel="GetLabsGroupLabel"
                visible="true">
           <button id="ColorsLabButton"
-                  getLabel="GetColorPickerButtonLabel"
+                  getLabel="GetLabel"
                   size="large"
-                  getImage="GetColorsLabImage"
-                  getSupertip="GetColorPickerButtonSupertip"
-                  onAction="ColorPickerButtonClick"/>
+                  getImage="GetImage"
+                  getSupertip="GetSupertip"
+                  onAction="OnAction"/>

```

Comment:
Idea: only need to provide the following methods
getLabel="GetLabel"
getImage="GetImage"
getSupertip="GetSupertip"
onAction="OnAction"